plugins/woocommerce/includes/class-wc-privacy.php

```diff
@@ -139,6 +139,7 @@ public function queue_cleanup_personal_data() {
 		self::$background_process->push_to_queue( array( 'task' => 'trash_pending_orders' ) );
 		self::$background_process->push_to_queue( array( 'task' => 'trash_failed_orders' ) );
 		self::$background_process->push_to_queue( array( 'task' => 'trash_cancelled_orders' ) );
+		self::$background_process->push_to_queue( array( 'task' => 'trash_refunded_orders' ) );
```

Comment:
@masteradhoc Should this be `anonymize_refunded_orders`?

Reply:
@albarin yes sorry first did the feature with trashing but then realized that we should do it same as the completed orders. Fixed it!